includes/admin/class-wc-payments-admin.php

```diff
@@ -438,4 +442,13 @@ private function is_tos_agreement_required() {
 	private static function is_account_overview_page_enabled() {
 		return get_option( '_wcpay_feature_account_overview' );
 	}
+
+	/**
+	 * Checks whether the grouped settings feature is enabled
+	 *
+	 * @return bool
+	 */
+	private static function is_grouped_settings_enabled() {
+		return get_option( 'wcpaydev_grouped_settings', '0' ) === '1';
```

Comment:
'yes'/'no' is the [convention](https://github.com/woocommerce/woocommerce/blob/23710744c01ded649d6a94a4eaea8745e543159f/tests/legacy/unit-tests/order/coupons.php#L62) usually used by option to designate boolean values that are actually stored as strings.

Reply:
The `update_option_from_checkbox` method in the dev tools helper stores these values as `'0'` or `'1'`, so I need to make the same check here